### PR TITLE
Remove unmaintained implementations and update table

### PR DESCRIPTION
**Pull request by a2379**
I think it would be good to remove unmaintained implementations to make room for new ones.

The removal criteria was no mailing list or repository activity in 3+ years, or the maintainer posts notice saying the implementation is unmaintained.

By that criteria Unsyntax is on the borderline of removal. The table is missing Cyclone and Jazz Scheme, both of which seem to be active.

### CODE REVIEW

**Comment by lassik** [comment]
Another issue is that the [SRFI  support table](https://docs.scheme.org/srfi/support/) is hard to read in general. It's just a grid of checkmarks with mouse-over pop-up tips.

We should probably add some text. Break it down with subheadings, categorize the SRFIs - things like that.

File: data/srfi-data.scm [hunk 1]
```diff
@@ -2171,12 +2171,13 @@
  (keywords data-structure)
  (draft-date "2023-11-14"))
 ((number 251)
- (status draft)
+ (status final)
```
Comment (arthurgleckler) [inline data/srfi-data.scm:2174]: Changes to this file should not be part of this change list.

Also, these changes reflect the current status of the file, so I'm not sure why they're here, anyway.

File: data/srfi-data.scm [hunk 2]
```diff
@@ -2171,12 +2171,13 @@
...
  (title "Mixing groups of definitions with expressions within bodies")
  (author "Sergei Egorov")
  (see-also 245)
  (keywords binding)
- (draft-date "2023-11-30"))
+ (draft-date "2023-11-30")
```
Comment (arthurgleckler) [inline data/srfi-data.scm:2179]: Same as above.

Comment (a2379) [reply data/srfi-data.scm:2179]: That file got updated while regenerating the table. I think it takes the values from the upstream srfi-data.scm.

I updated the title of this PR to indicate that the data is being updated as well.

Comment (lassik) [reply data/srfi-data.scm:2179]: I just pushed a commit to update srfi-data.scm.

Comment (arthurgleckler) [reply data/srfi-data.scm:2179]: Oh, I see.  Sorry about that.  I mistook this for the copy in srfi-common.  But since it's a copy, shouldn't it be in .gitignore?

Comment (lassik) [reply data/srfi-data.scm:2179]: It probably should be.

Comment (a2379) [reply data/srfi-data.scm]: I removed the intermediate files and added a .gitignore.

**Comment by a2379** [comment]
Added a note at the bottom of the table indicating the removal criteria, as per the discussion on the mailing list:
https://srfi-email.schemers.org/schemeorg/msg/24917138/

**Comment by lassik** [comment]
```
(p
     (b "* On removals: ")
     "Implementations will be removed from this table if either:"
     (br)
     (ol
      (li "There is no mailing list or repository activity for three or more years from the date this
table was generated.")
      (li "Their maintainer indicates that the implementation is unmaintained.")))
```

Let's write a complete policy (and a list of currently maintained implementations) and place it at some prominent central place on scheme.org.

**Comment by arthurgleckler** [comment]
I've merged it.  Would someone else please complete the process?  I don't have Racket installed, and don't have time right now.

**Comment by lassik** [comment]
I'll do it.
